### PR TITLE
Repair CI and UTs and v0.6.5

### PR DESCRIPTION
**Pull request by Ngoguey42**
- [X] Repair the UTs for the new version of pytest
- [X] Python version in CI `3.4.9/3.6.6/3.7.0` -> `3.5.7/3.6.6/3.7.0`
- [X] Create `changes.md` and `services.md`
- [x] setup circleci on earthcube-lab and repair all pipelines (we need to stop using the airware aws docker image registy)
- [x] Python version in CI `3.5.7/3.6.6/3.7.0` -> `3.6.12/3.7.9/3.8.5`
- [x] Edit README about drop of support for python `<3.6`
- [x] Restore codecov
- [x] Edit circleci and codecov badge in README
- [x] Bump versions

### CODE REVIEW

**Comment by Ngoguey42** [comment]
Someone with admin right on earthcube-lab has to authorize circle-ci https://circleci.com/docs/2.0/getting-started/

File: changes.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+---
+
+# 0.6.4
+Small bug fix when using numpy scalars
+
+---
+
+# 0.6.3
+Buzzard now have a documentation
```
Comment (azarz) [inline changes.md]:
```suggestion
Buzzard now has a documentation
```

File: services.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+List of external services used in the buzzard project
+- Github
+  - Admins: bound to the repo url
+- circle-ci
+  - Admins: bound to the repo url
+- codecov
+  - Admins: bound to the repo url
+- readthedoc
```
Comment (azarz) [inline services.md]:
```suggestion
- readthedocs
```

File: changes.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+## Public changes
+### Parameters update
+- Add `srs.wkt_of_file@center` now accepts coordinates
+
+---
+
+# 0.6.1
+## Public changes
+### Footprint coordinates rounding after reprojection
+Thanks to the new parameter of the `Footprint.move` method, opening a raster when using an `sr_work` in the `Dataset` constructor work much better than before.
```
Comment (azarz) [inline changes.md]:
```suggestion
Thanks to the new parameter of the `Footprint.move` method, opening a raster when using an `sr_work` in the `Dataset` constructor works much better than before.
```
Is it the parameter round_coordinates? Are there any caveats using this parameter?

File: changes.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+### New features
+- Can now decorate python function with an `Env` object
+- `fp.erode` and `fp.dilate` now accept more parameters to control the effect on each border
+- Add `fp.forward_conv2d`, `fp.backward_conv2d`, `fp.forward_convtranspose2d` and `fp.backward_convtranspose2d` to anticipate the effect of a convolution on the shape of a tensor
+
+### Bug fixes
+- Process pools managed by a `Dataset` are now joined without dead-lock (No UT yet for Pool management)
+- Fix `Env` to work with python threads (New UT)
+- Fix `fp.intersection(rotation='fit')` when `fp` has a non-north-up scale (No UT for non-north-up Footprints)
+- Fix bug when using and integer to select a raster band
```
Comment (azarz) [inline changes.md]:
```suggestion
- Fix bug when using an integer to select a raster band
```

**Comment by Ngoguey42** [comment]
Done 🎉

**Comment by Ngoguey42** [comment]
closes #5

**Pull request merged**